recipe/meta.yaml

```diff
@@ -1,22 +1,19 @@
 {% set name = "rubin-sim" %}
-{% set version = "0.6" %}
+{% set version = "0.6.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/lsst/rubin_sim/archive/refs/tags/{{ version }}.tar.gz
-  sha256: 6a6ada011f1b6dedd386a7017080972d3ab31d678457096c8515266cc3eb11b4
+  sha256: 257e99f21a85f17118018ba45dee679528a5a84f8346ea06f5cf93f638e344fc
 
 build:
   number: 0
-  skip: True  # [win]
+  skip: true  # [win]
   # this build cannot be noarch because of the scripts keyword in the setup.py
-  script: "{{ PYTHON }} -m pip install . -vv"
-  # this can be removed once https://github.com/lsst/rubin_sim/pull/92 is merged and release
-  script_env:
-    - SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
+  script: {{ PYTHON }} -m pip install . -vv
```

Comment:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv
  # this can be removed if upstream makes github releases and not just tags
  script_env:
    - SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
```